src/components/DashboardAnalytics/ProfitSharingAnalytics/ProfitSharingAnalytics.scss

```diff
@@ -13,6 +13,13 @@
     padding-left: 24px;
     width: 40%;
   }
+
+  .chartBox {
```

Comment:
@cwagner22  maybe there is no point keeping this style if all of it is commented.